en/sql/tuning.rst

```diff
@@ -2337,3 +2340,80 @@ The above **SELECT** query's plan is printed out as below; we can see "(sort lim
                    cost:  6 card 1000
         sort:  2 asc
         cost:  7 card 0
+
+.. _query-cache:
+
+QUERY CACHE
+===========
+
+The **QUERY_CACHE** hint can be used to enhance the performance for the query which is executed repeatedly. The query is cached in dedicated memory area and its results are also cached at the separated disk space. The hint is applied to SELECT query only; however for the following cases, the hint is not applied to the query:
```

Comment:
1. however for the following cases, -->however, (comma added is required)
2. the hint is not applied to the query: --> does not applied (passive change to active)

Reply:
I'd like to change the sentence 2 as below:
the hint is not applicable to the query and the hint is meaningless.

Reply:
okay, that would be good too